.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,16 @@
+name: SwiftLint
+on:
+  workflow_dispatch: {}
```

Comment:
This is for arbitrary run this action via Github Web UI. So you can trigger linting for any branch you want from a single click.